upwind-sensor/default.nix

```
@@ -129,7 +162,7 @@ in
       wantedBy = [ "multi-user.target" ];
```

```suggestion
      wantedBy = [ "multi-user.target" ];
      restartTriggers = [
        cfg.environment.etc."upwind/agent.yaml".source
        cfg.environment.etc."upwind/agent-hostconfig.yaml".source 
      ];
```

should make the agent restart on config chile changes